.changeset/curvy-numbers-sing.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': patch
+---
+
+Update toggle and checkbox to match fluent 2
```

Comment:
I would avoid mentioning fluent un changeset. What about this:
`Updating icons in toggle and checkbox components.`

Reply:
updated